client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -486,20 +492,25 @@ class EmailProvidersComparison extends React.Component {
 		};
 
 		return (
-			<PromoCard
-				isPrimary
-				title={ translate( 'Get your own @%(domainName)s email address', translateArgs ) }
-				image={ image }
-				className="email-providers-comparison__action-panel"
-			>
-				<p>
-					{ translate(
-						'Pick one of our flexible options to connect your domain with email ' +
-							'and start getting emails @%(domainName)s today.',
-						translateArgs
-					) }
-				</p>
-			</PromoCard>
+			<Fragment>
+				<HeaderCake onClick={ this.goBack }>
+					<h1>{ translate( 'Emails' ) }</h1>
```

Comment:
I don't think the `<h1>` wrapper is needed.

Reply:
Thank you, I saw it in that way somewhere else. I thought It was necessary.